docs/pages/billing/invoices-and-receipts.mdx

```diff
@@ -25,7 +25,7 @@ For example, an [Organization account's](/accounts/account-types/#organizations)
 
 To download and view an invoice for a billing period, go to [Receipts](https://expo.dev/settings/receipts) and:
 
-- Click the **Date** for the billing period corresponding to the invoice. You will be navigated to a page hosted by Stripe. As an example, the March 22, 2024 invoice below links to this page:
+- Click the **Date** for the billing period corresponding to the invoice. You will be navigated to a page hosted by Stripe. As an example, the March 22, 2025 invoice below links to this page:
```

Comment:
The date in this example is consistent with the date in the image below. If we want to update the date here, we should update the image below and in the next section.